api/tests/routes/backoffice/finance_test.py

```diff
@@ -196,6 +200,11 @@ def test_validate_incident(self, authenticated_client, force_debit_note):
             },
         )
 
+        final_deposit_value = db.session.query(sa.func.get_deposit_balance(deposit.id, False)).first()[0]
+
+        # To check that the deposit has been updated after the validation of the incident
+        assert final_deposit_value == first_deposit_value + booking_incident.booking.amount
```

Comment:
Comme au-dessus, je suggère plutôt : 
```suggestion
        # To check that the deposit has been updated after the validation of the incident
        assert user.wallet_balance == first_deposit_value + booking_incident.booking.amount
```
Je suis de plus d'accord avec @dbaty sur le fait de mettre explicitement un nombre, plutôt que faire un calcul : une mauvaise compréhension répercutée dans l'action et dans le résultat attendu donnera un test _Passed_, cachant une potentielle erreur, maintenant ou après des modifications futures.

Autre remarque : je préfère laisser `assert response.status_code == 200` AVANT toute assertion, car on préfère vérifier d'abord que la fonction n'a pas échoué. Peu importe la valeur du crédit si on a une erreur 500.